extensions/objects/models/objects.py

```diff
@@ -314,7 +314,22 @@ class MathLatexString(BaseObject):
 
     description = 'A LaTeX string.'
 
```

Comment:
Do we need to define the default value here?

Reply:
done